tests/ios/MASVS-STORAGE/MASTG-TEST-0052.md

```diff
@@ -100,3 +100,4 @@ Note: You may be asked to authenticate using the devices passcode or TouchID
 Save the output by adding `--json keychain.json` to this command
 Dumping the iOS keychain...
 Created                    Accessible                      ACL    Type      Account                    Service                                                        Data
+```
```

Comment:
Please go back to v1.5.0 and include the missing text from Document/0x06d-Testing-Data-Storage.md

Adapt the headings accordingly.

```suggestion
To use Objection to read the Keychain, execute the following command:

...

3. Cache can be also disabled by setting the Cache Policy to [.notAllowed](https://developer.apple.com/documentation/foundation/urlcache/storagepolicy/notallowed "URLCachePolicy notAllowed"). It will disable storing Cache in any fashion, either in memory or on disk.
```